coverage/lcov.info

```diff
@@ -0,0 +1,699 @@
+SF:lib\src\presentation\widget\bottom_loader.dart
```

Comment:
This file should not be added

Reply:
This is the autogenerated when you run the coverage

Reply:
whoops, removed it and added /coverage/ to gitignore